### PR TITLE
Me/foobar

### PR DESCRIPTION
**Pull request by meichstedt**
Added a foo

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/unterstein/upsource-test/pull/3" target="_blank">ReviewNinja

File: Foo.scala [hunk 1]
```diff
@@ -0,0 +1 @@
+case class Foo(name: String, index: Int, values: Map[String, Int])
```
Comment (unterstein) [inline Foo.scala]: What does the `index` do?

File: Bar.scala [hunk 2]
```diff
@@ -0,0 +1 @@
+case class Bar(id: String, foo: Foo)
```
Comment (unterstein) [inline Bar.scala]: Please add the empty line at the end !resolve

Comment (meichstedt) [inline Bar.scala]: Will do

Comment (meichstedt) [inline Bar.scala]: !fixed

File: Foo.scala (hunk 1 above)
Comment (meichstedt) [inline Foo.scala]: The index will be relevant for ordering later on. !resolved

Comment (unterstein) [inline Foo.scala]: !resolve  remove Foo to Foxy

File: Foxy.scala [hunk 3]
```diff
@@ -0,0 +1,6 @@
+/**
+  * Represents a foxy.
+  * @param id Id of the foxy
+  * @param values all associated foos by name
+  */
```
Comment (unterstein) [inline Foxy.scala:6]: !resolve muh?

File: Foo.scala (hunk 1 above)
Comment (unterstein) [inline Foo.scala]: Looks nice !resolve

Comment (meichstedt) [inline Foo.scala]: !resolved

File: Foxy.scala (hunk 3 above)
Comment (meichstedt) [inline Foxy.scala:6]: !resolved

File: Foxy.scala [hunk 4]
```diff
@@ -0,0 +1,7 @@
+/**
+  * Represents a foxy and it's associated foos.
+  * @param id Id of the foxy
+  * @param values all associated foos by name
+  * Use this with caution!
+  */
+case class Foxy(id: String, foosByName: Map[String, Foo])
```
Comment (unterstein) [inline Foxy.scala]: !fix empty lines please

Comment (meichstedt) [inline Foxy.scala]: There _is_ an empty line!

Comment (meichstedt) [inline Foxy.scala]: !fixed

**Comment by meichstedt** [comment]
… and even 4 years later, the question remains … why not just use github? 😂 

… and who the heck brought up phabricator and thus the excruciatingly dumb need to evaluate code review tooling?

**Pull request closed without merging**